src/rules/no-unused-modules.js

```diff
@@ -534,15 +534,15 @@ module.exports = {
       }
 
       // special case: export * from
-      const exportAll = exports.get(EXPORT_ALL_DECLARATION);
+      const exportAll = exports && exports.get(EXPORT_ALL_DECLARATION);
```

Comment:
Why not just return at https://github.com/import-js/eslint-plugin-import/pull/2990/files#diff-7d2908b010ec724c0f3a0d7615cb91f4673d765d52c6ba6607501e23470f2966R533

Reply:
I think we still need to report from line 567 to be shown:

```
context.report(
          node,
          `exported declaration '${value}' not used within other modules`,
        );
``` 
At the end

Reply:
The error console says the file has no exports then why it should report any export error? Or that just means an internal bug?

Reply:
The scenario is following:
1. create new file and define export it in
2. see eslint error that export is not used anywhere.

Without this PR we see eslint crash and no longer provide other checks and autofixes.
I assume this is internal bug mentioned in console warning, file *has* export, so this should be fixed somehow. till then this fix works as well